docs/src/logistic.md

```diff
@@ -39,10 +39,11 @@ saver = train.Saver()
 # Run training
 run(sess, global_variables_initializer())
 checkpoint_path = mktempdir()
-@info("Checkpoint files saved in $checkpoint_path")
+@info "Checkpoint files saved in $checkpoint_path"
```

Comment:
Why get rid of the parens?

Reply:
On second test the code works with or without parens.